src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+     class="search-bar search-bar_big"
+     data-qa= "big"
+      >
```

Comment:
The indentation seems to be a bit off for some of the tags.

Reply:
done! :)